js/views/aliases.js

```diff
@@ -1,5 +1,5 @@
 /**
- * ownCloud - Mail
+ * Mail
```

Comment:
Is there a reason of why we are not simply replacing that with `Nextcloud - Mail` ?! 😁 Does not matter anyway - but yeah 😅